example/testNewLib.m

```diff
@@ -0,0 +1,15 @@
+system1 = neqsim.thermo.system.SystemSrkEos(280,10);
```

Comment:
Her må du kalle Matlab-funksjon thermo for å lage System. Visst ikkje må pathNeqSim kallast osv.

Reply:
Også fint om du skriv kva versjonsnummber neqsim-jar har når du oppdaterar